packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -12,6 +12,8 @@ import 'binding.dart';
 import 'box.dart';
 import 'debug.dart';
 import 'object.dart';
+import 'proxy_box.dart';
```

Comment:
Hm, sliver depending on proxy box is a bit weird, but ok. we already depend on box, i guess.